backend/src/main/resources/application.properties

```diff
@@ -4,3 +4,6 @@ spring.graphql.schema.locations=classpath:/specs/
 spring.graphql.schema.file-extensions=graphql
 # TODO-2506: Server port is currently duplicated in config.yml but actually read from here; the config source should be unified
 server.port=8000
+spring.servlet.multipart.enabled=true
+spring.servlet.multipart.max-file-size=5MB
+spring.servlet.multipart.max-request-size=10MB
```

Comment:
❓ Depending on how exactly this is calculated, this might result in only one part being allowed, since a request always contains some extra data.